<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>新纪录的类型:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>&lt;NONE&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>衣服</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>食品</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>化妆品</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>家具</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>记录的完好度:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>&lt;NONE&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>九成新及以上</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>七-八成新</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>五-六成新</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>四成新及以下</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>记录的所有者:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>&lt;NONE&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>        Graphics:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>215</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>SURE!</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>66</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QDockWidget" name="dockWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>4</y>
          <width>58</width>
          <height>351</height>
         </rect>
        </property>
        <widget class="QWidget" name="dockWidgetContents_2">
         <widget class="listwidget" name="listWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>21</y>
            <width>41</width>
            <height>321</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>29</y>
          <width>291</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="yifu"/>
        <widget class="QWidget" name="shiwu"/>
        <widget class="QWidget" name="huazhuangpin"/>
        <widget class="QWidget" name="jiaju"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="Menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>listwidget</class>
   <extends>QListWidget</extends>
   <header>listwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
